src/rf/js/src/home/components/modals.js

```diff
@@ -293,8 +327,12 @@ var UploadModal = React.createBackboneClass({
             }
         }
 
-        if (!layerData.area) {
-            errors.area = ['Please enter an area.'];
+        if (layerData.capture_start && !layerData.capture_end) {
+            errors.capture_end.push('Please enter a date.');
+        }
+
+        if (layerData.capture_end && !layerData.capture_start) {
+            errors.capture_start.push('Please enter a date.');
         }
```

Comment:
I'm not sure where this validation logic for start/end dates originated. I don't see how these dates are any different than other readonly fields that appear on the layer/image detail panel.